src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.FreeBSD.cs

```diff
@@ -20,28 +20,32 @@ internal static string GetProcPath(int processId)
             return Interop.Process.GetProcPath(processId);
         }
 
-        private static ProcessInfo CreateProcessInfo(int pid)
+        internal static ProcessInfo? CreateProcessInfo(int pid, string? processNameFilter = null)
         {
             // Negative PIDs aren't valid
             if (pid < 0)
             {
                 throw new ArgumentOutOfRangeException(nameof(pid));
             }
 
-            ProcessInfo procInfo = new ProcessInfo()
-            {
-                ProcessId = pid
-            };
-
             // Try to get the task info. This can fail if the user permissions don't permit
             // this user context to query the specified process
             ProcessInfo iinfo = Interop.Process.GetProcessInfoById(pid);
```

Comment:
why 'iinfo'

Reply:
no idea. I didn't write or change it here.